README.md

```diff
@@ -7,6 +7,10 @@ The Kubernetes Template Project is a template for starting new projects in the G
 - a `CONTRIBUTING.md` outlining how to contribute to the project
 - the `code-of-conduct.md` from this repo, which outlines community behavior and the consequences of breaking the code
 - a `LICENSE` which must be Apache 2.0 for code projects, or Creative Commons 4.0 for documentation repositories
+- a `SECURITY_CONTACTS` with the contact points for the Product Security Team 
```

Comment:
nit: trailing whitespace here and below.

```console
$ git show --check --oneline origin/pr/18
533cc7d (origin/pr/18) add SECURITY_CONTACTS
README.md:10: trailing whitespace.
+- a `SECURITY_CONTACTS` with the contact points for the Product Security Team 
README.md:12: trailing whitespace.
+  [Embargo Policy](https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#embargo-policy) 
```

Reply:
You guys can we just merge this my time is a lot more valuable than fixing this stuff...

Reply:
> You guys can we just merge this my time is a lot more valuable than fixing this stuff...

I'm the peanut gallery, not a maintainer, so feel free to ignore.